landoapi/spec/swagger.yml

```diff
@@ -84,14 +121,53 @@ paths:
           schema:
             allOf:
               - $ref: '#/definitions/Error'
+  /landings/{landing_id}:
+    get:
+      description: |
+        Request the status of the landing job.
+      parameters:
+        - name: landing_id
+          in: path
+          type: string
+          description: |
+            The id of the landing to return
+          required: true
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/Landing'
+        404:
+          description: Landing does not exist
+          schema:
+            allOf:
+              - $ref: '#/definitions/Error'
+        default:
+          description: Unexpected error
+          schema:
+            allOf:
+              - $ref: '#/definitions/Error'
 definitions:
-  Transplant:
+  Landing:
     type: object
     properties:
+      id:
+        type: integer
+        description: |
+          The id of the Landing
       request_id:
         type: integer
         description: |
-          The id of the request in Transplant service
+          The id of the Request in Transplant service
```

Comment:
I don't think we should expose the transplant request_id to the public. What do you think?